pyannote/audio/transforms/pipeline.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from torch.nn import Sequential
+
+
+class TransformsPipe(Sequential):
```

Comment:
Apart from being able to access a transform by index, what is the benefit of this compared to `torch_audiomentations.Compose`? . 

Wouldn't it be better for long term support to add what you think is missing to `Compose` directly?

Reply:
There was more to this but I removed it. I'll create a PR to get it to work with Compose from audiomentations